release_notes.md

```diff
@@ -3,6 +3,8 @@
 - My change description (#PR)
 -->
 
+* Add extensions and concurrency sections to the SyncTriggerPayload (ArmCacheEnabled = true) (#7863)
```

Comment:
Let's remove this ArmCacheEnabled comment. This feature should work regardless of ArmCacheEnabled state, right?

Reply:
Yes. will do it.

Reply:
Done